### ISSUE TITLE
All tables are being considered a layout table

### ISSUE THREAD

**Issue by renekoch** [qualweb/wcag-techniques#1]
I cant get the wcag to treat table as "data tables" 

html below fails with  Rule ID: [F46](https://www.w3.org/WAI/WCAG21/Techniques/failures/F46)

to test :
```html
<table summary="asdasd sada sdasd asda dsa"><caption>asdasdsadasdsd</caption>
    <thead>
        <tr>
            <th colspan="3" id="th1">th 1</th>
            <th id="th2">th 2</th>
            <th id="th3">th 3</th>
            <th id="th4">th 4</th>
        </tr>            
    </thead>
    <tbody>
        <tr>
            <th rowspan="8" id="th5" headers="th1">th 5</th>
            <th rowspan="4" id="th6" headers="th1 th5">th 6</th>
            <th rowspan="2" id="th7" headers="th1 th5 th6">th 7</th>
            <td headers="th2 th5 th6 th7">td 1</td>
            <td headers="th3 th5 th6 th7">td 2</td>
            <td headers="th4 th5 th6 th7">td 3</td>
        </tr>
        <tr>
            <td headers="th2 th5 th6 th7">td 4</td>
            <td headers="th3 th5 th6 th7">td 5</td>
            <td headers="th4 th5 th6 th7">td 6</td>
        </tr>
        <tr>
            <th rowspan="2" id="th8" headers="th1 th5 th6">th 8</th>
            <td headers="th2 th5 th6 th8">td 7</td>
            <td headers="th3 th5 th6 th8">td 8</td>
            <td headers="th4 th5 th6 th8">td 9</td>
        </tr>
        <tr>
            <td headers="th2 th5 th6 th8">td 10</td>
            <td headers="th3 th5 th6 th8">td 11</td>
            <td headers="th4 th5 th6 th8">td 12</td>
        </tr>
        <tr>
            <th rowspan="4" id="th9" headers="th1 th5">th 9</th>
            <th rowspan="2" id="th10" headers="th1 th5 th9">th 10</th>
            <td headers="th2 th5 th9 th10">td 13</td>
            <td headers="th3 th5 th9 th10">td 14</td>
            <td headers="th4 th5 th9 th10">td 15</td>
        </tr>
        <tr>
            <td headers="th2 th5 th9 th10">td 16</td>
            <td headers="th3 th5 th9 th10">td 17</td>
            <td headers="th4 th5 th9 th10">td 18</td>
        </tr>
        <tr>
            <th rowspan="2" id="th11" headers="th1 th5 th9">th 11</th>
            <td headers="th2 th5 th9 th11">td 19</td>
            <td headers="th3 th5 th9 th11">td 20</td>
            <td headers="th4 th5 th9 th11">td 21</td>
        </tr>
        <tr>
            <td headers="th2 th5 th9 th11">td 22</td>
            <td headers="th3 th5 th9 th11">td 23</td>
            <td headers="th4 th5 th9 th11">td 24</td>
        </tr>
    </tbody>
</table>
```
